media-gfx/iscan/iscan-3.62.0.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools desktop flag-o-matic udev xdg-utils
+
+DESCRIPTION="EPSON Image Scan v3 for Linux"
+HOMEPAGE="https://support.epson.net/linux/en/imagescanv3.php"
```

Comment:
I suggest:
```
HOMEPAGE="https://support.epson.net/linux/en/imagescanv3.php https://gitlab.com/utsushi/utsushi"
```

Reply:
Corrected

Reply:
I'm not sure if it's appropriate to have 'gitlab.com/utsushi/utsushi' here.

There is 'gitlab.com/utsushi/imagescan', but that only tracks upstream ('support.epson.net').  Changes are then applied over that in 'gitlab.com/utsushi/utsushi', but this is not what this ebuild seems to be about.

See also: https://gitlab.com/utsushi/imagescan/-/issues/3

Reply:
But this ebuild also has `make_desktop_entry utsushi "Image Scan"`, which means that `utsushi` command would be run by `Image Scan` menu entry.
Maybe both gitlab homepages should be mentioned...
```
HOMEPAGE="https://support.epson.net/linux/en/imagescanv3.php https://gitlab.com/utsushi/imagescan https://gitlab.com/utsushi/utsushi"
```

Reply:
Yeah, the upstream (Epson) build also uses 'utsushi' for the binary name.  It's all a bit confusing to say the least.